OrbitGl/EventTrack.cpp

```diff
@@ -19,6 +19,10 @@ EventTrack::EventTrack(TimeGraph* a_TimeGraph) : Track(a_TimeGraph) {
 std::string EventTrack::GetTooltip() const { return "Left-click and drag to select samples"; }
 
 void EventTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
+  if (IsEmpty()) {
+    return;
+  }
+
```

Comment:
This will change the current behavior for the all the event tracks, which I don't think we want. If a thread has no sampling event, then we won't draw the colored event track. That event track is nice to identify the threads by color, I'd prefer keeping the original behavior.

We should early out only for the case of the AllTracePointsFakeTid.

Reply:
Of course! Thank you!